src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -9,16 +10,37 @@ import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
 import { Button } from "../Components/Button"
 import { CenteringContainer, Container } from "../Components/Containers"
 import { Divider } from "../Components/Divider"
+import { MarkdownRenderer } from "../Components/MarkdownRenderer"
 import { Timer } from "../Components/Timer"
 
+// import { SelectMaxBid_sale_artwork } from "__generated__/SelectMaxBid_sale_artwork.graphql"
+import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.graphql"
+
+// interface MinimumNextBid {
+//   display: string
+//   cents: number
+// }
```

Comment:
With this and the commented-out import, let's take those out if we don't need them.